GrandCentralBoard/Widgets/Harvest/HarvestWidgetSettings.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  HarvestWidgetSettings.swift
+//  GrandCentralBoard
+//
+//  Created by Karol Kozub on 2016-04-11.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import Decodable
+
+
+struct HarvestWidgetSettings: Decodable {
+    let accessToken: String
+    let refreshToken: String
+    let expirationDate: NSDate
+    let refreshInterval: NSTimeInterval
+
+    static func decode(json: AnyObject) throws -> HarvestWidgetSettings {
+        let accessToken = try json => "accessToken" as! String
+        let refreshToken = try json => "accessToken" as! String
+        let expirationTime = (try json => "expirationTime" as! NSNumber).doubleValue
```

Comment:
Don't use `as!`, instead write throwable routine to cast and throw if it failed.